### PR TITLE
Example

### PR DESCRIPTION
**Pull request by alexcb**

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.10
```
Comment (vladaionescu) [inline Earthfile:1]: Should we use alpine for faster builds?

**Review by vladaionescu — APPROVED** [review]
Looks great!

File: Earthfile (hunk 1 above)
Comment (alexcb) [reply Earthfile:1]: I wanted to, but protoc requires `__strftime_l` which isn't included under alpine's glibc lib, so I ended up using ubuntu which is definitely more bloated but got me working faster.

Comment (vladaionescu) [reply Earthfile:1]: :-( makes sense

**Pull request merged**